test/src/unit-capi-consolidation.cc

```diff
@@ -4036,6 +4182,56 @@ int ConsolidationFx::get_meta_num(const char* path, void* data) {
   return 1;
 }
 
+int ConsolidationFx::get_array_meta_files_callback(
```

Comment:
Also here, shall we just return the vector?

Reply:
Same issue as above... this is a callback routine for get_array_meta_files_callback.